miniscope_io/stream_daq.py

```diff
@@ -79,6 +79,7 @@ def __init__(
         self.preamble = self.config.preamble
         self._buffer_npix: Optional[List[int]] = None
         self._nbuffer_per_fm: Optional[int] = None
+        self.terminate = multiprocessing.Value('b', False)
```

Comment:
probably use an `Event` or something for this, i think `Value` has a decent amount of overhead (but i could be wrong)